frontend/src/screens/Relay.tsx

```
@@ -84,71 +135,132 @@ export const Relay = () => {
+        </Text>
+      </Group>
+
+      <Box ml="32">
+        <Text color="dimmed" size="md" mb="md">
+          Make sure to port forward the OrPort on your router if behind NAT. See
+          your router's documentation for specific instructions on how to set up
+          port forwarding or visit{' '}
+          <a
+            href="https://www.wikihow.com/Set-Up-Port-Forwarding-on-a-Router"
+            target="_blank"
+          >
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)